RestaurantManagementSystem/RestaurantManagementSystem/src/Infrastructure/RegisterRestaurant/RequestRestaurant.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
```

Comment:
this should be the RegistrationRequestRepo and have the update also in the same class